### DISCUSSION TITLE
WHERE with AND and OR

### DISCUSSION THREAD

**Discussion by faridprogrammer** [maxtoroq/DbExtensions#40] [Q&A]
How to generate query with WHERE clause that have multiple parts?
In [this](http://stackoverflow.com/questions/31035822/dbextensions-where-with-and-and-or) question developer wants to generate query like this

```
  SELECT * 
  FROM myTable
  WHERE (Field1 LIKE '%a%' OR Field1 LIKE '%b%')
     AND (Field2 LIKE '%a%' OR Field2 LIKE '%b%')
```

Is there simple way to do it?

**Comment by maxtoroq** [comment]
Answered on StackOverflow.
